src/main/java/org/jenkinsci/plugins/codesonar/models/json/CodeSonarChartData.java

```diff
@@ -0,0 +1,26 @@
+package org.jenkinsci.plugins.codesonar.models.json;
+
+import java.util.List;
+
+/**
+ * @author aseno
+ *
+ */
+public class CodeSonarChartData {
```

Comment:
This class name `CodeSonarChartData` suggests that the class is for generic chart data.  However, it is clearly for `CodeSonarAnalysisWarningCount` data.

I think you should rename this class to `CodeSonarWarningCountChartData`, or consider making the class generic.